CHANGES.md

```diff
@@ -39,13 +39,16 @@ New tabs now open on top.
 [#1178](https://github.com/cylc/cylc-ui/pull/1178) - Improve appearance of
 Table view.
 
+[#1043](https://github.com/cylc/cylc-ui/pull/1043) - The GScan sidebar is
+now resizeable.
+
 ### Fixes
 
 [#1164](https://github.com/cylc/cylc-ui/pull/1164) -
 Fix the mean run time column in the table view.
 
-[#1167](https://github.com/cylc/cylc-ui/pull/1168) - Prevent mutations menu
-jumping when expanded.
+[#1167](https://github.com/cylc/cylc-ui/pull/1168) - Prevent command menu from
+jumping after a delay when clicking "see more".
```

Comment:
Reminder that users probably won't know what a "mutation" is, hence I prefer the term "command"